project_restorer/src/main/scala/PlutoCoreMessageProcessor.scala

```diff
@@ -43,11 +43,17 @@ class PlutoCoreMessageProcessor(mxsConfig:MatrixStoreConfig)(implicit mat:Materi
    } yield result
   }
 
-  def handleStatusMessage(updateMessage: ProjectUpdateMessage):Future[Either[String, Json]] = {
+
+  def sendToMessageExchange(msgs: Seq[OnlineOutputMessage], routingKey: String, framework: MessageProcessingFramework) = {
+    framework.bulkSendMessages("Nearline.project.e", routingKey, msgs)
+  }
```

Comment:
TBH this method is probably not required, you can just call `framework.bulkSendMessages` at line 56 in `handleStatusMessage`

Reply:
What does `Nearline.project.e` refer to?

Reply:
I changed it to: `nearline.project-restorer.associatedmsgs` instead. Do you have another name you prefer?

Reply:
hmm it's the exchange to send the messages on? Personally I would just use the "output exchange" from the framework instance.   The routing key allows us to disambiguate what the messages are from.